snuba/datasets/events/__init__.py

```diff
@@ -405,3 +406,11 @@ def get_extensions(self) -> Mapping[str, QueryExtension]:
 
     def get_prewhere_keys(self) -> Sequence[str]:
         return ['event_id', 'issue', 'tags[sentry:release]', 'message', 'environment', 'project_id']
+
+    def split_query(
+            self,
+            query_func: SplitQueryFunc,
+            *args: Any,
```

Comment:
Why do you need to allow *args and **kwargs ?
This function should not receive anything other than a query_func